gettsim/taxes/zu_verst_eink/eink.py

```diff
@@ -190,65 +97,12 @@ def eink_rente_zu_verst(
     return rente_ertragsanteil * 12 * sum_ges_rente_priv_rente_m
 
 
-def bruttolohn_m_tu(bruttolohn_m: FloatSeries, tu_id: IntSeries) -> FloatSeries:
-    """Sum monthly wages in tax unit.
-
-    Parameters
-    ----------
-    bruttolohn_m
-        See basic input variable :ref:`bruttolohn_m <bruttolohn_m>`.
-    tu_id
-        See basic input variable :ref:`tu_id <tu_id>`.
-
-    Returns
-    -------
-    FloatSeries with sum of monthly wages per tax unit.
-    """
-    return bruttolohn_m.groupby(tu_id).sum()
-
-
-def kapitaleink_brutto_m_tu(
-    kapitaleink_brutto_m: FloatSeries, tu_id: IntSeries
-) -> FloatSeries:
-    """Aggregate monthly capital income on tax unit level.
-
-    Parameters
-    ----------
-    kapitaleink_brutto_m
-        See basic input variable :ref:`kapitaleink_brutto_m <kapitaleink_brutto_m>`.
-    tu_id
-        See basic input variable :ref:`tu_id <tu_id>`.
-
-    Returns
-    -------
-
-    """
-    return kapitaleink_brutto_m.groupby(tu_id).sum()
-
-
-def vermiet_eink_m_tu(vermiet_eink_m: FloatSeries, tu_id: IntSeries) -> FloatSeries:
-    """Aggregate monthly rental income on tax unit level.
-
-    Parameters
-    ----------
-    vermiet_eink_m
-        See basic input variable :ref:`vermiet_eink_m <vermiet_eink_m>`.
-    tu_id
-        See basic input variable :ref:`tu_id <tu_id>`.
-
-    Returns
-    -------
-
-    """
-    return vermiet_eink_m.groupby(tu_id).sum()
-
-
 def sum_eink_ohne_kapital(
-    eink_selbst: FloatSeries,
-    eink_abhängig_beschäftigt: FloatSeries,
-    eink_vermietung: FloatSeries,
-    eink_rente_zu_verst: FloatSeries,
-) -> FloatSeries:
+    eink_selbst: float,
+    eink_abhängig_beschäftigt: float,
+    vermiet_eink: float,
```

Comment:
Did we decide to turn this around? I see no reason to change the name (but might have suggested it previously, of course), especially since the official thing is "Einkommen aus Vermietung und Verpachtung", if I am not mistaken.

Reply:
I also don't remember.. changed it to `eink_vermiet` now

Reply:
Why not leave it? We decided on it that way for a reason...

Reply:
Now I am confused. I thought you prefer `eink_vermiet`. I don't have a strong preference here.